tests/expression_converter_/test_op_expressions.py

```diff
@@ -0,0 +1,205 @@
+import datetime
+from uuid import UUID
+
+from bson import Decimal128
+from django.test import SimpleTestCase
+
+from django_mongodb_backend.query_conversion.expression_converters import convert_expression
+
+
+class TestBaseExpressionConversionCase(SimpleTestCase):
+    CONVERTIBLE_TYPES = {
+        "int": 42,
+        "float": 3.14,
+        "decimal128": Decimal128("3.14"),
+        "boolean": True,
+        "NoneType": None,
+        "string": "string",
+        "datetime": datetime.datetime.now(datetime.UTC),
+        "duration": datetime.timedelta(days=5, hours=3),
+        "uuid": UUID("12345678123456781234567812345678"),
+    }
+
+    def assertConversionEqual(self, input, expected):
+        result = convert_expression(input)
+        self.assertEqual(result, expected)
+
+    def assertNotOptimizable(self, input):
+        result = convert_expression(input)
+        self.assertIsNone(result)
+
+    def test_non_dict_expression(self):
+        expr = ["$status", "active"]
+        self.assertNotOptimizable(expr)
+
+    def test_empty_dict_expression(self):
+        expr = {}
+        self.assertNotOptimizable(expr)
+
+    def test_non_convertible(self):
+        expr = {"$gt": ["$price", 100]}
+        self.assertNotOptimizable(expr)
```

Comment:
Tests shouldn't appear in a test case since they'll be run redundantly on each subclass.